style.css

```diff
@@ -0,0 +1,454 @@
+  display: flex;
+  align-items: center;
+  gap: 25px;
+  cursor: pointer;
+}
+
+#phone_navbar > div > img {
+  cursor: pointer;
+  width: 132px;
+  height: 30px;
+}
+
+#user {
+  color: #fdc7c3;
+}
+
+#menu {
+  color: #fb9b93;
+  font-size: larger;
+}
+
+#nav_bar_desktop {
+  display: none;
+}
+
+/* phone main */
+
+main {
+  background-color: #f4f5f6;
+  height: auto;
+  margin-top: 20px;
+}
+
+.section_cont {
+  background-color: white;
+}
+
+.featured {
+  row-gap: 15px;
+}
+
+.topstory {
+  row-gap: 15px;
+}
+
+.background1 {
+  background-image: url(https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711616/donald-trump.webp?w=397&amp;h=265&amp;q=90&amp;f=d759057143d283aa0ea0ed13fba5450a);
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 250px;
+}
+
+.background1 > p {
+  margin-bottom: 0;
+  padding: 5px 15px 6px;
+  background-color: rgba(255, 255, 255, 0.9);
+  font-weight: bold;
+  font-size: large;
+  color: red;
+}
+
+.background2 {
+  background-image: url(https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711649/biden-speaks-delaware.webp?w=591&h=364&f=400256ef3c52b699d57e5db01061d213);
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 350px;
+}
+
+.background2 > p {
+  margin-bottom: 0;
+  padding: 5px 15px 6px;
+  background-color: rgba(255, 255, 255, 0.9);
+  font-weight: bold;
+  font-size: large;
+  color: red;
+}
+
+.section2 {
+  row-gap: 25px;
+}
+
+.section2_under {
+  margin-top: 10px;
+  row-gap: 5px;
+}
+
+.small_img {
+  width: 100%;
+}
+
+.img_litle {
+  width: 95px;
+  height: 95px;
+}
+
+#part1 > div {
+  border-top: 1px solid #ddd;
+  padding-top: 10px;
+}
+
+#part2 > div {
+  border-top: 1px solid #ddd;
+  padding-top: 10px;
+}
+
+.text_div {
+  column-gap: 20px;
+}
+
+#line {
+  height: 3px;
+  border-top: 1px solid #d9d9d9;
+  border-bottom: 1px solid #d9d9d9;
+}
+
+.debate_cont {
+  row-gap: 20px;
+}
+
+.col3_title {
+  color: black;
+  font-weight: bold;
+  font-size: 20px;
+}
+
+.col3_text {
+  font-weight: bold;
+  color: #555;
+}
+
+.small_icon {
+  width: 63px;
+  height: 63px;
+}
+
+.col2_title {
+  color: #818181;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+.col2_title:hover {
+  text-decoration: none;
+  color: #818181;
+}
+
+/* phone footer */
+
+footer {
+  width: 100%;
+  height: auto;
+  background-color: white;
+}
+
+.nav_top {
+  margin-bottom: 20px;
+}
+
+#logo_bot {
+  width: 250px;
+}
+
+.nav_top2 {
+  column-gap: 15px;
+}
+
+.copyright {
+  font-size: 13px;
+  color: #666;
+  margin: auto;
+}
+
+.nav_top2 > a > i {
+  font-size: 25px;
+  color: #666;
+}
+
+.nav_bot1 {
+  column-gap: 20px;
+  margin-top: 10px;
+}
+
+#title_editors {
+  color: #666;
+  font-weight: bold;
+}
+
+.nav_bot2 {
+  gap: 10px;
+}
+
+.nav_bot2 > ul {
+  column-gap: 20px;
+}
+
+.nav_bot2 > ul > li > a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#hide {
+  display: none;
```

Comment:
- [ ] use `d-none` Bootstrap class for this, also, don't mark these as an ID because you might use it for multiple elements.